tests/unit/ui/components/TestShowMoreCard.js

```diff
@@ -40,7 +40,7 @@ describe(__filename, () => {
 
   it('truncates the contents if they are too long', () => {
     const root = render({ children: 'Hello I am description' });
-    root.truncateToMaxHeight({ clientHeight: 101 });
+    root.truncateToMaxHeight({ clientHeight: MAX_HEIGHT + 1 });
```

Comment:
What is the extra 1px needed for?

Reply:
@muffinresearch , to test `(contents.clientHeight >= MAX_HEIGHT)` this condition.
Initially we were doing it manually `101 px` but then @willdurand suggested to export the variable.